manifests/backup/mariabackup.pp

```diff
@@ -0,0 +1,89 @@
+# mariadb::backup::mariabackup
+#
+# Manage MariaDB mariabackup Backup
+#
+# @summary Manage MariaDB mariabackup Backup
+#
+# @example
+#   include mariadb::backup::mariabackup
+
+class mariadb::backup::mariabackup (
+  $mariabackup_package_name = $mariadb::params::backup_package_name,
+  $backupuser              = undef,
+  $backuppassword          = undef,
+  $backupdir               = $mariadb::params::backupdir,
+  $backupmethod            = 'mariabackup',
+  $backupdirmode           = '0700',
+  $backupdirowner          = $mariadb::params::user,
+  $backupdirgroup          = $mariadb::params::group,
```

Comment:
I'm not sure about using the mysql user/group. The cron runs as root and the xtrabackup uses root: https://github.com/puppetlabs/puppetlabs-mysql/blob/master/manifests/backup/xtrabackup.pp#L14

Reply:
Changed to root